x/ccv/consumer/types/params.go

```diff
@@ -101,3 +139,21 @@ func validateProviderFeePoolAddrStr(i interface{}) error {
 	// Otherwise validate as usual for a bech32 address
 	return ccvtypes.ValidateBech32(i)
 }
+
+func validateConsumerRedistributionFraction(i interface{}) error {
+	str, ok := i.(string)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+	dec, err := sdk.NewDecFromStr(str)
+	if err != nil {
+		return err
+	}
+	if !dec.IsPositive() {
```

Comment:
From the name this function doesn't allow 0. If that is true, we should change that.

Reply:
Nice catch! Turns out there's an `IsNegative` method